apps/extension/postcss.config.js

```diff
@@ -1 +1 @@
-module.exports = require('@penumbra-zone/ui/postcss.config');
+export { default } from '@penumbra-zone/ui/postcss.config.js';
```

Comment:
Since `@penumbra-zone/ui` (and every other package in this monorepo) is now a `"type": "module"`, the Node resolver treats an import without an extension differently, expecting it to be listed in the module's `package.json` `files` setting (or something?). Adding `.js` makes it explicit that we are importing that file specifically.